experiments/cli.py

```diff
@@ -0,0 +1,494 @@
+import os, os.path
+import errno
+import requests
+import base64
+import time
+import json
+from pathlib import Path
+
+import click
+
+from experiments.sdk.evolvclient.config import EvolvConfig
+from experiments.sdk.evolvclient.client import EvolvClient
+
+
+APPLICATION_JSON = 'application/json'
+APPLICATION_YAML = 'application/yaml'
+
+EVOLV_AUTH_FILE = '/tmp/.evolv/auth.json'
```

Comment:
Make this cross platform using tempfile? Think Windows has it under c:\windows...Think you can check some env variable to get the OS tmp directory